generators/docker-compose/templates/_consul.yml

```diff
@@ -11,7 +11,7 @@ services:
         command: consul agent -dev -ui -client 0.0.0.0 -config-dir /consul-conf
 
     consul-config-loader:
-        image: jhipster/consul-config-loader
+        image: jhipster/consul-config-loader:v0.1.0
```

Comment:
Can we change `jhipster/consul-config-loader:v0.1.0` to a CONST here: [generator-constants.js](https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-constants.js)

So it will be easier to update Docker images, instead of looking everywhere in the code